api/src/main/java/com/github/retrooper/packetevents/protocol/world/chunk/ChunkBitMask.java

```diff
@@ -0,0 +1,48 @@
+package com.github.retrooper.packetevents.protocol.world.chunk;
+
+import com.github.retrooper.packetevents.manager.server.ServerVersion;
+import com.github.retrooper.packetevents.wrapper.PacketWrapper;
+
+import java.util.BitSet;
+
+public class ChunkBitMask {
```

Comment:
~~this class can be made package-private, its only used for encoding/decoding `LightData`~~ nevermind

this should still be marked as `@ApiStatus.Internal` and probably also have a private constructor